app/formats/json/PanoramaHistoryFormats.scala

```diff
@@ -0,0 +1,21 @@
+package formats.json
+
+import play.api.libs.json.{JsPath, Reads}
+import play.api.libs.functional.syntax._
+
+object PanoramaHistoryFormats {
```

Comment:
Let's mess with the names of some of these to shorten them and make code easier to read (and in some cases for clarity/consistency).
1. Let's just use "pano" instead of "panorama" almost anywhere
2. We use the IDs for the panoramas in lots of other places, and the two names we use are `pano_id` or `gsv_panorama_id` (and their camelcase counterpars); let's use `pano_id` for everything here instead of `pano` or `currentId` (`currentId` could be `currentPanoId` though). 
3. `IndividualPanoHistory` still sounds like it contains the entire history... What about `PanoDate`? Or if we include the current pano's info at the top level of `PanoHistorySubmission` instead of within the history list, we could use `OldPanoDate`/`PrevPanoDate`?
4. To match convention, the field names that we read from JS should be snake_case and the Scala should be camelCase. So the Scala looks good, but the JSON should be like `visited_timestamp` instead of `visitedTimestamp`